Gemfile

```diff
@@ -276,7 +276,7 @@ group :development, :test, :sandbox do
   gem 'capybara'
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
-  gem 'webdrivers'
+  gem 'webdrivers', '~> 5.2.0'
```

Comment:
@200455939-yashu Why are we using this specific version of webdrivers?

Reply:
We could leave it at ```gem 'webdrivers', '~> 5.2'```

Reply:
It is breaking Rspec test cases. On an update to this version of 'webdrivers', issues are resolved.

Reply:
That is good to know!

We shold still keep with version ```gem 'webdrivers', '~> 5.2'``` to get upgrades.